packages/dialtone-tokens/postcss/dialtone-tokens.cjs

```diff
@@ -73,17 +73,18 @@ function boxShadows (shadowDeclarations, Declaration) {
           return `var(${shadowVar}${shadowNumber}-offset-x) var(${shadowVar}${shadowNumber}-offset-y) var(${shadowVar}${shadowNumber}-blur) var(${shadowVar}${shadowNumber}-spread) var(${shadowVar}${shadowNumber}-color)${isInset ? ' inset' : ''}`;
         }).join(', ');
 
-      shadowDeclarations.at(-1).after(new Declaration({ prop: shadowVar, value }));
+      shadowDeclarations.at(0).after(new Declaration({ prop: shadowVar, value }));
```

Comment:
Why this was -1 previously? 🤔

Reply:
I had to reverse the array to get it to work correctly in the shadow parsing function after recent changes, and I thought if I put it at the end of the array it would output at the bottom, but actual array order doesn't seem to affect the order postcss uses to output (probably uses a linked list or something internally). This will now output after the shadow declarations instead of the top of the list, so it makes more logical sense when you are looking at the file.

No functional difference since order of css vars doesn't matter logic wise.